product-matrix.json

```diff
@@ -44,6 +32,19 @@
       "dartPluginVersion": "202.4357.23",
       "sinceBuild": "202.4357.23",
       "untilBuild": "202.*"
+    },
+    {
```

Comment:
Similarly, I'd move this block of json up so that the platform versions are roughly in order.

Reply:
Done